frontend/libs/core/src/components/navbar/navbar.component.html

```diff
@@ -1,22 +1,22 @@
-<nav class="fixed navbar bg-white flex z-[100]">
+<nav class="fixed navbar bg-base-100 flex z-[100]">
   <!-- navbar left -->
   <div class="navbar-start">
     <a routerLink="/">
-      <img src="/assets/logo/mnext.svg" class="h-[3.5rem]" alt="mnext logo" />
+      <smart-energy-logo />
     </a>
   </div>
 
   <!-- navbar center -->
   <div class="navbar-center flex">
     <ul class="menu menu-horizontal px-1 space-x-1">
       <li *ngIf="isLoggedIn && isManager">
-        <a routerLink="/users" routerLinkActive="bg-gray-100">Gebruikers</a>
+        <a routerLink="/users" routerLinkActive="active">Gebruikers</a>
```

Comment:
Vind het gebruiken van de secondary kleur voor de geselecteerde pagina wat minder mooi. Deze gebruiken we in de applicatie vooral voor belangrijke knoppen, maar de active pagina is een knop die niet zo veel aandacht moet vragen.

Reply:
Ik had daar problemen mee want hij flikkert elke keer naar blauw als je op link klikt daarom had ik ook die kleur gekozen. Weet jij mischien hoe ik dat kan fixen?

Reply:
Als je terug zet hoe de navbar eerst was en het volgende in de `style.css` zet dan worden ze niet meer blauw:

```
.navbar-center .menu li a:active {
 @apply !bg-gray-300 !text-gray-900;
}
```

Reply:
Heb nu veranderd. heb nu wel bg-base-200 want liever geen hardcoded kleuren en deed anders ook raar met darkmode.